src/tests/ftest/cart/SConscript

```diff
@@ -40,7 +40,7 @@
 import os
 import daos_build
 
-SIMPLE_TEST_SRC = ['threaded_client.c',
+SIMPLE_TEST_SRC = ['threaded_client.c', 'dual_iface_server.c',
```

Comment:
General question.  If we have a situation where test3 depends on functionality tested in test2, and test2 depends on functionality tested in test1, then should we list the order of execution as "test1, test2, test3"?  Or doesn't the order matter for our testing.

Reply:
the order does not matter for our testing; further the list here is just list of tests to compile, they wont be auto-run